From 2f6e676eb52353049b6a517895c8c8e5697148bb Mon Sep 17 00:00:00 2001
From: Francisco Boni <boboniboni@gmail.com>
Date: Sun, 26 Dec 2021 22:09:27 -0300
Subject: [PATCH 22/22] ctest parallel 1

---
 test/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/CMakeLists.txt b/test/CMakeLists.txt
index 72bca3f9..6e9d3f56 100644
--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -3,7 +3,7 @@ ENABLE_TESTING()
 include ( FluidUnitTest )
 
 # first define the test target, used by the macros below
-add_custom_target(check COMMAND ${CMAKE_CTEST_COMMAND} -C $<CONFIG>  --output-on-failure)
+add_custom_target(check COMMAND ${CMAKE_CTEST_COMMAND} -C $<CONFIG>  --parallel 1 --output-on-failure)
 
 
 ## add unit tests here ##
-- 
2.34.1

